src/main/java/org/motechproject/mots/dto/LocationPreviewDto.java

```diff
@@ -25,6 +25,16 @@
   @JsonInclude(Include.NON_NULL)
   private String parent;
 
+  @Setter
+  @Getter
+  @JsonInclude(Include.NON_NULL)
+  private String district;
```

Comment:
why it's not named districtName? you will not need to add mappings in LocationMapper then